task/buildah-proxy-oci-ta/0.3/buildah-proxy-oci-ta.yaml

```diff
@@ -0,0 +1,933 @@
+---
+apiVersion: tekton.dev/v1
+kind: Task
+metadata:
+  name: buildah-proxy-oci-ta
```

Comment:
Is it supposed to be just a buildah task with a generic proxy?

Reply:
A a buildah task with the generic proxy (Indy sidecar) and the domain proxy (Unshare proxy). See the image I added to the PR description.